.github/workflows/package.yml

```diff
@@ -155,37 +171,77 @@ jobs:
       image-name: argilla/argilla-server
       dockerfile: docker/Dockerfile
       readme: README.md
-      platforms: linux/amd64,linux/arm64
+      platforms: linux/amd64
     secrets: inherit
 
-  build_quickstart_docker_image:
-    name: Build Argilla quickstart docker image
+  build_quickstart_docker_image_linux_amd64:
+    name: Build Argilla quickstart docker image linux/amd64
     uses: ./.github/workflows/build-push-docker.yml
-    needs: build_server_docker_image
+    needs: build_server_docker_image_linux_amd64
     if: |
       !cancelled() &&
-      needs.build_server_docker_image.result == 'success'
+      needs.build_server_docker_image_linux_amd64.result == 'success'
     with:
       download-python-package: false
       image-name: argilla/argilla-quickstart
       dockerfile: docker/quickstart.Dockerfile
       readme: docker/quickstart.README.md
-      platforms: linux/amd64,linux/arm64
+      platforms: linux/amd64
       build-args: |
-        ARGILLA_VERSION=${{ needs.build_server_docker_image.outputs.version }}
+        ARGILLA_VERSION=${{ needs.build_server_docker_image_linux_amd64.outputs.version }}
     secrets: inherit
 
+  # Build of arm64 Docker images only when release, as we need QEMU virtualization and it's very slow
+  build_server_docker_image_linux_arm64:
+    name: Build Argilla server docker image linux/arm64
+    uses: ./.github/workflows/build-push-docker.yml
+    needs:
+      - build_python_package
+      - run_unit_tests
+    if: |
+      !cancelled() &&
+      github.event_name == 'release' &&
+      needs.build_python_package.result == 'success' &&
+      !contains(needs.*.result, 'failure') &&
+      !contains(needs.*.result, 'cancelled')
+    with:
+      download-python-package: true
+      image-name: argilla/argilla-server
+      dockerfile: docker/Dockerfile
+      readme: README.md
+      platforms: linux/arm64
+    secrets: inherit
+
+  build_quickstart_docker_image_linux_arm64:
```

Comment:
can't we combine this using a template combined with  build_quickstart_docker_image_linux_amd64?